runestone/__main__.py

```diff
@@ -30,6 +30,8 @@ def init():
     conf_dict['author'] = click.prompt("Your Name ", default=getpass.getuser())
     conf_dict['project_title'] = click.prompt("Title for this project ", default="Runestone Default")
     conf_dict['python3'] = click.prompt("Use Simple Python3 Semantics ", default="false")
+    conf_dict['language'] = click.prompt("Locale ", default="en")
+    conf_dict['hide_control_captions'] = click.prompt("Hide control captions? ", type=bool, default=False)
```

Comment:
I can't tell what the hide_control_captions option is supposed to be for.  Can you tell me a little about its purpose?